account_invoice_show_currency_rate/models/res_currency.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCurrency(models.Model):
+    _inherit = "res.currency"
+
+    invoice_currency_rate_display = fields.Selection(
+        [
+            ("unit_per_company", "Unit per Company Currency"),
+            ("company_per_unit", "Company Currency per Unit"),
+        ],
+        default="unit_per_company",
+        string="Invoice Rate Display",
```

Comment:
Not a strong suggestion.

```suggestion
    invoice_rate_display_type = fields.Selection(
        [
            ("normal", "Unit per Company Currency"),
            ("inverse_rate", "Company Currency per Unit"),
        ],
        default="normal",
        string="Invoice Rate Display Type",
```